<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8">

    <title>CAP Regression</title>

    <meta name="author" content="Xi (Rossi) LUO">

    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">

    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

    <link rel="stylesheet" href="reveal.js_4_5/dist/reset.css">
    <link rel="stylesheet" href="reveal.js_4_5/dist/reveal.css">
    <link rel="stylesheet" href="reveal.js_4_5/dist/theme/white.css" id="theme">



    <!-- Theme used for syntax highlighting of code -->
    <link rel="stylesheet" href="reveal.js_4_5/plugin/highlight/tomorrow-night-blue.css">
    <!--
    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.2.0/styles/default.min.css">
-->
    <link rel="stylesheet" href="./CSS/rossisimple2.css" id="theme">
    <!--    <link rel="stylesheet" href="./css/brightRoom.css" id="theme">-->

    <!--    Cover Flow   -->
    <link rel="stylesheet" type="text/css" href="JSLibrary/coverflowjs/coverflow.css" />

    <script src="JSLibrary/jquery-3.2.1.js"></script>
    <script src="JSLibrary/coverflowjs/coverflow.min.js"></script>

    <!-- Coverflow styles -->
    <style>
        .ui-coverflow {
            position: relative;
            padding-bottom: 12px;
            margin-bottom: 5px;
        }

        .ui-coverflow>img,
        .ui-coverflow>figure {
            height: 440px !important;
            width: auto;
            min-width: 440px !important;
        }
    </style>
</head>

<body>
    <div class="reveal">
        <!--      place holder for credits -->
        <credit></credit>
        <!-- Any section element inside of this container is displayed as a slide -->
        <!-- Outline:  -->
        <div class="slides">
            <section>
                <h2><strong>Covariance Outcome Modelling via <highlight>C</highlight>ovariate <highlight>A</highlight>ssisted
                        <highlight>P</highlight>rincipal (<highlight>CAP</highlight>) Regression
                    </strong></h2>
                <br>
                <h3>Xi (Rossi) <strong>LUO</strong></h3>
                <br>
                <div width="100%">
                    <p class="inlinemiddle" style="display: inline-block; text-align: right; width:43%; vertical-align: middle;">
                        <medium><strong>The University of Texas</strong>
                            <br>Health Science Center
                            <br>School of Public Health
                            <br>Dept of Biostatistics
                            <br>and Data Science
                            <br>ABCD Research Group&nbsp;
                        </medium>
                    </p>
                    <img class="noborder inlinemiddle" style="display: inline-block;
                          vertical-align: middle;" src="Media/ABCDgroup/ABCDLogo4Color5.png" height="195px">
                </div>
                <br>
                <p>
                    <medium><strong>EcoSta 2023, Waseda University, Tokyo, Japan</strong>
                        <br>August 1, 2023
                    </medium>
                </p>
                <p>
                    <small> Funding: NIH R01MH126970, R01EB022911; NSF/DMS 1557467</small>
                </p>
            </section>


            <section>
                <h2>Major Collaborators</h2>
                <div style="display:block;">
                    <div class="fbox"> <img src="./Media/Personnel/YiZhao17.jpg" style="border-radius: 50%" alt="Yi Zhao" width="150" height="150">
                        <p class="fbox">Yi Zhao
                            <br>
                            <small>Indiana Univ</small>
                        </p>
                    </div>
                    <span style="display:inline-block; width: 50px;"></span>
                    <div class="fbox"> <img src="./Media/Personnel/BingkaiWang.jpg" style="border-radius: 50%" alt="Bingkai Wang" width="150" height="150">
                        <p class="fbox">Bingkai Wang
                            <br>
                            <small>UMich Biostat</small>
                        </p>
                    </div>
                </div>
                <p></p>
                <p></p>
                <div style="display:block;">
                    <div class="fbox"> <img src="./Media/Personnel/rayli.jpg" style="border-radius: 50%" alt="Stewart Mostofsky" width="150" height="150">
                        <p class="fbox">
                            <medium>Chiang-shan Ray Li</medium>
                            <br>
                            <small>Yale Univ</small>
                        </p>
                    </div>
                    <span style="display:inline-block; width: 50px;"></span>
                    <div class="fbox"> <img src="./Media/Personnel/stewartmostofsky.jpg" style="border-radius: 50%" alt="Stewart Mostofsky" width="150"
                            height="150">
                        <p class="fbox">
                            <medium>Stewart Mostofsky</medium>
                            <br>
                            <small>Johns Hopkins Medicine</small>
                        </p>
                    </div>
                    <span style="display:inline-block; width: 50px;"></span>
                    <div class="fbox"> <img src="./Media/Personnel/briancaffo.jpg" style="border-radius: 50%" alt="Brian Caffo" width="150" height="150">
                        <p class="fbox">Brian Caffo
                            <br>
                            <small>Johns Hopkins Biostat</small>
                        </p>
                    </div>
                </div>

            </section>

            <section>
                <h1>Slides viewable on web: <br>
                    <emph>bit.ly</emph>/<highlight>capuh23</highlight>
                </h1>
            </section>

            <!--            <div class="source">NSF</div>-->

            <section>
                <h2>Statistics/Data Science Focuses</h2>
                <div id="venn"></div>
                <script src="bower_components/d3/d3.v4.min.js"></script>
                <script src="bower_components/venn.js/venn.js"></script>
                <script>
                    // define sets and set set intersections
                    var sets = [{ sets: ['This Talk'], "label": "This Talk", "size": 200 },
                    { sets: ['Theory'], "label": "Theory", "size": 100 },
                    { sets: ['Method'], "label": "Method", "size": 100 },
                    { sets: ['Model'], "label": "Model", "size": 100 },
                    { sets: ['Application'], "label": "Application", "size": 100 },
                    { sets: ['Science'], "label": "Science", "size": 100 },
                    { sets: ['This Talk', 'Theory'], "size": 10 },
                    { sets: ['This Talk', 'Method'], "size": 20 },
                    { sets: ['This Talk', 'Model'], "size": 45 },
                    { sets: ['This Talk', 'Application'], "size": 20 },
                    { sets: ['This Talk', 'Science'], "size": 5 }];

                    var chart = venn.VennDiagram()
                        .width(1000).height(600);
                    var customOrdering = { "This Talk": 1, "Theory": 2, "Mehtod": 3, "Model": 4, "Application": 5, "Science": 6 }
                    chart.orientationOrder(function (a, b) { return customOrdering[a.setid] - customOrdering[b.setid] })

                    d3.select("#venn").datum(sets).call(chart);
                    d3.select("#venn .venn-circle path").style("fill-opacity", 0.7)
                    d3.selectAll("#venn text").style("stroke-width", 0.5).style("stroke", "#000");
                </script>
            </section>

            <section data-state="fMRI">
                <h2>Resting-state fMRI Networks</h2>
                <div class="twocol inlinemiddle" style="width: 35%">
                    <img src="Media/fMRI/fMRIScannerNSFLivescienceCut.png" alt="" class="noborder">
                </div>
                <div class="twocol inlinemiddle" style="width: 60%">
                    <ul>
                        <li>fMRI measures brain activities over time</li>
                        <li>Resting-state: "do nothing" during scanning</li>

                    </ul>
                </div>
                <br>
                <br>
                <div class="twocol inlinemiddle" style="width: 60%">
                    <ul>
                        <li>Brain networks constructed using <emph>cov/cor</emph> matrices of time series</li>
                    </ul>
                </div>
                <div class="twocol inlinemiddle" style="width: 35%">
                    <img src="Media/fMRI/brainFunctionConnTimeSeries.png" alt="" class="noborder">
                </div>

                <style>
                    .fMRI credit:after {
                        content: "Image source: NSF";
                    }
                </style>
            </section>

            <section data-background-iframe="Media/fMRI/ThreeJSBrainMaskColor.html">
                <img src="Media/Brain/brainICBMcut.png" alt="" class="noborder topright" width="20%">
                <!--                <h2>fMRI Data</h2>-->
                <!--                <iframe id="brainCube" src="Media/fMRI/ThreeJSBrainMaskColor.html" frameborder="0" width="800" height="600" name="brainCube"></iframe>-->
                <p style="margin-bottom: 650px;"></p>
                <p><strong>fMRI data</strong>: blood-oxygen-level dependent (BOLD) signals from each
                    <highlight>cube</highlight>. Each subject: observe matrix regions $\times$ time
                </p>
                <aside class="notes">
                    <ul>
                        <li>visualization is big challenge, too</li>
                        <li>Lego brain of real brain</li>
                        <li>red cube is a voxel</li>
                        <li>track activity from each cube at each time vector</li>
                        <li>dimension: address temporal separately</li>
                    </ul>
                </aside>
            </section>


            <section>
                <h2>Brain Networks</h2>
                <ul>
                    <li>Brain network analysis: an emerging trend <cite>Park, Friston, Science, 2013</cite></li>
                    <li>Estimating brain networks from data
                        <ul>
                            <li>Many methodological frameworks: cov, inv cov, DCM, Granger,ICA, frenquency, dynamic...
                            </li>
                            <li>Our group worked on graphical models <cite>Cai et al, 2001, Liu, L, 2015</cite>, DCM
                                <cite>Cao et al, 2019</cite>
                            </li>
                        </ul>
                    </li>
                    <li>Also on quantifying network info flow <cite>Zhao, L, 2019, Zhao L, 2021</cite></li>
                    <li>On utilization of networks for clustering <cite>Bunea et al, 2020</cite></li>
                    <li>Tree-like networks and regression <cite>Wang et al, 2022; Zhao et al, In Press</cite></li>
                    <!--                    <li>Excellent work by others on prediction or explaining phenotypes using networks <cite>   Chen, Jiang, Kong, Li, Ogden...</cite> </li>-->
                    <li>This talk: explaining network differences</li>
                </ul>
            </section>


            <section data-state="flaticon">
                <h2>Motivating Example</h2>
                <img src="Media/Network/brainNetworkbyAge.png" alt="">
                <p></p>
                <p>Brain network connections vary by covariates (e.g. age/sex)</p>
                <br>
                <p>
                    <emph>Goal</emph>: model how covariates change network connections
                </p>
                <style>
                    .flaticon credit:after {
                        content: "Image source: flaticon.com";
                    }
                </style>
            </section>

            <section>
                <h2>Our Idea</h2>
                $$\textrm{function}(\textbf{graph}) = \textbf{age}\times \beta_1 + \textbf{sex}\times \beta_2 + \cdots
                $$
            </section>

            <section>
                <h2>Other Related Applications</h2>
                <ul>
                    <li>Differences in genetic networks
                        <ul>
                            <li>Networks by disease/age/sex groups</li>
                            <li>Single cell sequencing</li>
                        </ul>
                    </li>
                    <li>Covariance modeling
                        <ul>
                            <li>Temporal changes in covariance matrices</li>
                            <li>Spatial-temporal</li>
                        </ul>
                    </li>
                    <li>Theme: <highlight>signals</highlight> in the <emph>second</emph> moment, low/no
                        signals in
                        the first moment</li>
                </ul>
            </section>

            <section>
                <h2>Mathematical Problem</h2>
                <ul>
                    <li>Given $n$ (semi-)positive matrices, $\Sigma_i\in \real^{p\times p}$</li>
                    <li>Given $n$ corresponding vector covariates, $x_i \in \real^{q}$</li>
                    <li>Find function $g(\Sigma_i) = x_i \beta$, $i=1,\dotsc, n$</li>
                    <li>In essense, <emph>regress positive matrices on vectors</emph>
                    </li>
                </ul>
            </section>

            <section>
                <h2>Other Extensions</h2>
                <br>
                <ul>
                    <li>Linear model formulation, and thus very flexible</li>
                    <li>Easily extended to big data with 1 million brain regions, Brain and Behavior</li>
                    <li>High dim cov (later), Electronic J of Stat</li>
                    <li>Many other extensions under review or preparation: longitudinal, mixed effects, ...</li>
                </ul>
            </section>


            <section>
                <h2>Some Related Problems</h2>
                <ul>
                    <li>Heterogeneous regression or weighted LS:
                        <ul>
                            <li>Usually for scalar variance $\sigma_i$, find $g(\sigma_i) = f(x_i)$</li>
                            <li>Goal: to improve efficiency, not to interpret $x_i \beta$</li>
                        </ul>
                    </li>
                    <li>Covariance models <cite>Anderson, 73; Pourahmadi, 99; Hoff, Niu, 12; Fox, Dunson, 15; Zou,
                            17</cite>
                        <ul>
                            <li>Model $\Sigma_i = g(x_i)$, sometimes $n=i=1$</li>
                            <li>Goal: better models for $\Sigma_i$</li>
                        </ul>
                    </li>
                    <li>Multi-group PCA, (inverse) covariance estimation/testing <cite>Flury,
                            84, 88; Boik 02; Hoff 09;
                            Franks,
                            Hoff, 16, Guo et al, 11; Tsai et al, 22; Cai et al, 13</cite>
                        <ul>
                            <li>No regression model, cannot handle vector $x_i$</li>
                            <li>Goal: find common/uncommon parts of multiple (inv) $\Sigma_i$</li>
                        </ul>
                    </li>
                    <li>Tensor-on-scalar regression <cite>Li, Zhang, 17; Sun, Li, 17</cite>
                        <ul>
                            <li>No guarantees for positive matrix outcomes</li>
                        </ul>
                    </li>
                </ul>
            </section>

            <section>
                <h1>CAP for Low-dimensional Cov Outcomes</h1>
            </section>

            <section>
                <h2>Naive: Massive Edgewise Regressions</h2>
                <ul>
                    <li>Intuitive method by mostly neuroscientists</li>
                    <li>Try $g_{j,k}(\Sigma_i) = \Sigma_{i}[j,k] = x_i \beta$</li>
                    <li>Repeat for all $(j,k) \in \{1,\dotsc, p\}^2$ pairs</li>
                    <li>Essentially $O(p^2)$ regressions for each connection</li>
                    <li>Limitations:
                        <ul>
                            <li>multiple testing $O(p^2)$</li>
                            <li>failure to accout for dependencies between regressions</li>
                            <li>low signal to noise ratio elementwise, spectral modeling may be more robust</li>
                        </ul>
                    </li>
                </ul>
            </section>

            <section>
                <h2>Our CAP in a Nutshell</h2>
                <br>
                <h2>
                    <strong>$\mbox{Modified PCA}(\Sigma_i) = x_i \beta$</strong>
                </h2>
                <ul>
                    <li>Essentially, we aim to turn unsupervised PCA to a supervised PCA</li>
                    <li>Ours differs from existing PCA methods:
                        <ul>
                            <li>Supervised PCA <cite>Bair et al, 06</cite> models <emph>scalar-on-vector</emph>
                            </li>
                        </ul>
                    </li>
                </ul>
            </section>


            <section>
                <h1>Model and Method</h1>
            </section>


            <section>
                <h2>Model</h2>
                <ul>
                    <li>Find principal direction (PD) $\gamma \in \real^p$, such that:
                        $$ \log({\gamma}^\top\Sigma_{i}{\gamma})=\beta_{0}+x_{i}^\top{\beta}_{1}, \quad i =1,\dotsc, n$$
                    </li>
                </ul>
                <img src="Media/CAP/cap2dcontour.png" style="width: 35%" alt="">
                <p>Example (p=2): PD1 largest variation in $\Sigma_i$ but not related to $x$</p>
                <p>
                    <emph>PCA</emph> selects <emph>PD1</emph>, <highlight>Ours</highlight> selects <highlight>PD2
                    </highlight>
                </p>
            </section>

            <section>
                <h2>Advantages</h2>
                <ul>
                    <li>Scalability: potentially for $p \sim 10^6$ or larger</li>
                    <li>Interpretation: covariate assisted PCA
                        <ul>
                            <li>Turn <highlight>unsupervised</highlight> PCA into <highlight>supervised</highlight>
                            </li>
                        </ul>
                    </li>
                    <li>Sensitivity: target those covariate-related variations
                        <ul>
                            <li>
                                <highlight>Covariate assisted</highlight> SVD?
                            </li>
                        </ul>
                    </li>
                    <li>Potential applications in other big data problems besides fMRI</li>
                </ul>
            </section>

            <section>
                <h2>Method</h2>
                <ul>
                    <li>MLE with constraints:
                        $$\scriptsize \begin{eqnarray}\label{eq:obj_func}
                        \underset{\boldsymbol{\beta},\boldsymbol{\gamma}}{\text{minimize}} &&
                        \ell(\boldsymbol{\beta},\boldsymbol{\gamma}) :=
                        \frac{1}{2}\sum_{i=1}^{n}(x_{i}^\top\boldsymbol{\beta}) \cdot T_{i}
                        +\frac{1}{2}\sum_{i=1}^{n}\boldsymbol{\gamma}^\top \Sigma_{i}\boldsymbol{\gamma} \cdot
                        \exp(-x_{i}^\top\boldsymbol{\beta}) , \nonumber \\
                        \text{such that} && \boldsymbol{\gamma}^\top H \boldsymbol{\gamma}=1
                        \end{eqnarray}$$
                    </li>
                    <li>No meaningful solutions without constraints</li>
                    <li>Two obvious constriants:
                        <ul>
                            <li>C1: $H = I$</li>
                            <li>C2: $H = n^{-1} (\Sigma_1 + \cdots + \Sigma_n) $</li>
                        </ul>
                    </li>
                    <li>The most natural C1 usually does <emph>not</emph> work!</li>
                </ul>
            </section>


            <section>
                <h2>Choice of $H$</h2>
                <div class="thm"><strong>Proposition:</strong> When (C1) $H=\boldsymbol{\mathrm{I}}$ in the optimization
                    problem, for any fixed $\boldsymbol{\beta}$, the solution of $\boldsymbol{\gamma}$ is the
                    eigenvector corresponding to the minimum eigenvalue of matrix
                    $$ \sum_{i=1}^{n}\frac{\Sigma_{i}}{\exp(x_{i}^\top\boldsymbol{\beta})} $$
                </div>
                <br>
                <ul>
                    <li>C1 leads to <emph>small</emph> eigen values (potential noises)</li>
                    <li>Will focus on the constraint (C2)</li>
                </ul>
            </section>

            <section>
                <h2>Algoirthm</h2>
                <ul>
                    <li>Iteratively update $\beta$ and then $\gamma$ </li>
                    <li>Prove explicit updates (see our papers)</li>
                    <li>Extension to multiple $\gamma$:
                        <ul>
                            <li>After finding $\gamma^{(1)}$, we will update $\Sigma_i$ by removing its effect</li>
                            <li>Search for the next PD $\gamma^{(k)}$, $k=2, \dotsc$</li>
                            <li>Impose the orthogonal constraints such that $\gamma^{k}$ is orthogonal to all
                                $\gamma^{(t)}$ for $t\lt k$</li>
                        </ul>
                </ul>
            </section>

            <section>
                <h2>Theory for $\beta$</h2>
                <div class="thm"><strong>Theorem:</strong>
                    Assume $\sum_{i=1}^{n}x_{i}x_{i}^\top/n\rightarrow Q$ as $n\rightarrow\infty$. Let
                    $T=\min_{i}T_{i}$, $M_{n}=\sum_{i=1}^{n}T_{i}$, under the true $\boldsymbol{\gamma}$, we have
                    \begin{equation}
                    \sqrt{M_{n}}\left(\hat{\boldsymbol{\beta}}-\boldsymbol{\beta}\right)\overset{\mathcal{D}}{\longrightarrow}\mathcal{N}\left(\boldsymbol{\mathrm{0}},2
                    Q^{-1}\right),\quad \text{as } n,T\rightarrow\infty,
                    \end{equation}
                    where $\hat{\boldsymbol{\beta}}$ is the maximum likelihood estimator when the true
                    $\boldsymbol{\gamma}$ is known.
                </div>
            </section>

            <section>
                <h2>Theory for $\gamma$</h2>
                <div class="thm"><strong>Theorem:</strong>
                    Assume $\Sigma_{i}=\Gamma\Lambda_{i}\Gamma^\top$, where
                    $\Gamma=(\boldsymbol{\gamma}_{1},\dots,\boldsymbol{\gamma}_{p})$ is an orthogonal matrix and
                    $\Lambda_{i}=\mathrm{diag}\{\lambda_{i1},\dots,\lambda_{ip}\}$ with $\lambda_{ik}\neq\lambda_{il}$
                    ($k\neq l$), for at least one $i\in\{1,\dots,n\}$. There exists $k\in\{1,\dots,p\}$ such that for
                    $\forall~i\in\{1,\dots,n\}$,
                    $\boldsymbol{\gamma}_{k}^\top\Sigma_{i}\boldsymbol{\gamma}_{k}=\exp(x_{i}^\top\boldsymbol{\beta})$.
                    Let $\hat{\boldsymbol{\gamma}}$ be the maximum likelihood estimator of $\boldsymbol{\gamma}_{k}$ in
                    Flury, 84. Then assuming that the assumptions are satisfied, <br><br> $\hat{ \boldsymbol{\beta}}$
                    from
                    our
                    algorithm is $\sqrt{M_{n}}$-consistent estimator of $\boldsymbol{\beta}$.
                </div>
            </section>


            <section>
                <h1>Simulations</h1>
            </section>


            <section>
                <img src="Media/CAP/simuCompareTable.png" alt="">
                <br>
                <p>PCA and common PCA do <emph>not</emph> find the first principal direction, because they don't
                    consider covariates
                </p>
            </section>


            <section>
                <h1>Resting-state fMRI</h1>
            </section>

            <section>
                <h2>Regression Coefficients</h2>
                <br>
                <div class="threecol inlinemiddle">
                    <img src="Media/CAP/hcpRestCoef_Page_1.png" alt="">
                    <p>Age</p>
                </div>
                <div class="threecol inlinemiddle">
                    <img src="Media/CAP/hcpRestCoef_Page_2.png" alt="">
                    <p>Sex</p>
                </div>
                <div class="threecol inlinemiddle">
                    <img src="Media/CAP/hcpRestCoef_Page_3.png" alt="">
                    <p>Age*Sex</p>
                </div>
                <br>
                <br>
                <ul>
                    <li>
                        <highlight>Ours above</highlight>: <emph>significant</emph> network differences due to age, sex
                        and their
                        interactions
                    </li>
                    <li>Naive massive edgewise regression: <emph>no</emph> statistical significant changes in cov
                        entries</li>
                </ul>
            </section>


            <section>
                <h2>Brain Map of $\gamma$</h2>
                <img src="Media/CAP/loadingMap.png" style="width: 100%" alt="">
            </section>


            <section>
                <h2>High-Dim Cov Extensions</h2>
                <ul>
                    <li>Voxel level (super-high dim) cov matrices <cite>Zhao et al, 2020</cite>
                        <ul>
                            <li>Raw cov: $10^6 \times 10^6$=Trillions of cov elements</li>
                            <li>Decompose data/networks via ICA/PCA</li>
                            <li>Explain netowrk diff on reduced dim</li>
                            <li>Reconstruct brain network maps at the voxel level</li>
                        </ul>
                    </li>
                    <li>High dimensional cov <cite>Zhao et al, 2021</cite>
                        <ul>
                            <li>Joint shrinkage <cite>Ledoit, Wolf, 2004</cite> of multiple cov</li>
                            <li>Optimum in theory by our joint shrinkage</li>
                            <li>Method/theory also work with joint shrinkage</li>
                        </ul>
                    </li>

                </ul>
            </section>

            <section>
                <h1>CAP for High-dimensional Cov Outcomes</h1>
            </section>


            <section>
                <h2>Challenges in High-dim</h2>
                <ul>
                    <li>Sample covariance not full rank in high-dim: when sample size $T_i$ &#8810; variable size $p$
                    </li>
                    <li>$p$ fixed in the previous theory</li>
                    <li>Sample cov is a poor estimator in high dim</li>
                    <li>Eigenvalues even more dispersed</li>
                    <li>Regularization <cite>Ledoit, Wolf 04</cite> less optimal</li>
                </ul>
            </section>

            <section>
                <h2>L-W Cov Shrinkage</h2>
                <ul>
                    <li>By Ledoit and Wolf (2004), over 2800 citations so far</li>
                    <li>Given sample cov $S$, find estimator $\Sigma^*$ by
                        $$
                        \begin{eqnarray}
                        \underset{\mu,\rho}{\text{minimize}} &&
                        \mathbb{E}\left\| \Sigma^{*}- S \right\|^{2}
                        \nonumber \\
                        \text{such that} && \Sigma^{*}=\rho\mu\boldsymbol{\mathrm{I}}+(1-\rho)S
                        \end{eqnarray}
                        $$
                    </li>
                    <li>$\rho$ and $\mu$ can be estimated consistently, and thus $\Sigma^*$ consistently</li>
                    <li>Limitations: <emph>not</emph> handling multiple covariance matrices jointly, <emph>not</emph>
                        using the
                        covariate info
                    </li>
                </ul>
            </section>

            <section>
                <h2>Our Contributions</h2>
                <ul>
                    <li>New <emph>joint</emph> shrinkage estimator for <emph>multiple</emph> covariances</li>
                    <li>Incorporate covariate info in regression</li>
                    <li>
                        <emph>Minimum</emph> quadratic risk asymptotically among all linear combinations, while L-W is
                        suboptimal
                    </li>
                    <li>Also provide consistent estimator for CAP</li>
                </ul>
            </section>

            <section>
                <h2>Covariate Dependent CAP Shrinkage</h2>
                <ul>
                    <li>Propose CAP shrinkage by
                        $$
                        \begin{eqnarray}
                        \underset{\mu,\rho}{\text{minimize}} &&
                        \frac{1}{n}\sum_{i=1}^{n}\mathbb{E}\left\{\gamma^\top\Sigma_{i}^{*}\gamma-\exp(
                        x_{i}^\top\beta)\right\}^{2}
                        \nonumber \\
                        \text{such that} && \Sigma_{i}^{*}=\rho\mu\boldsymbol{\mathrm{I}}+(1-\rho)S_{i}, \quad
                        \text{for } i=1,\dots,n.
                        \end{eqnarray}
                        $$
                    </li>
                    <li>Prove a theorem for solving the optimization and an estimator for $\Sigma_i^*$
                        $$
                        S_i^* = \hat{f}(\gamma, x_i, \beta) \boldsymbol{\mathrm{I}} + \hat{g}(\gamma, x_i, \beta)
                        S_i
                        $$
                        where $S_i^* $ is a consistent estimator, $\hat{f}$ and $\hat{g}$ are computed by explicit
                        formulas from data
                    </li>
                </ul>
            </section>

            <section>
                <h2>CS-CAP Algorithm</h2>
                <img src="Media/CAP/high_dim_cap_algo.png" alt="" width="90%">
                <p>
                    Iteratively update $S^*$, $\gamma$, $\beta$ until cconvergence
                </p>
            </section>


            <section>
                <div class="thm"><strong>Theorem:</strong>
                    Assume $(\gamma, \beta)$ is given. With a fixed $n\in\mathbb{N}^{+}$, for <emph>any sequence</emph>
                    of linear
                    combinations $\{\hat{\Sigma}_{i}\}_{i=1}^{n}$ of the identity matrix and the sample covariance
                    matrix, where the combination coefficients are constant over $i\in\{1,\dots,n\}$, the estimator
                    $ S_{i}^{*}$ verifies:
                    $$ \tiny
                    \begin{equation}
                    \lim_{T\rightarrow\infty}\inf_{T_{i}\geq
                    T}\left[\frac{1}{n}\sum_{i=1}^{n}\mathbb{E}\left\{\gamma^\top\hat{\Sigma}_{i}\gamma-\exp(\bx_{i}^\top
                    \beta)\right\}^{2}-\frac{1}{n}\sum_{i=1}^{n}\mathbb{E}\left\{\gamma^\top
                    S_{i}^{*}\gamma-\exp(\bx_{i}^\top \beta)\right\}^{2}\right]\geq
                    0.
                    \end{equation}
                    $$
                    In addition, every sequence of $\{\hat{\Sigma}_{i}\}_{i=1}^{n}$ that performs as well as
                    $\{ S_{i}^{*}\}_{i=1}^{n}$ is <emph>identical</emph> to $\{ S_{i}^{*}\}_{i=1}^{n}$ in the limit:
                    $$
                    \tiny
                    \begin{equation}
                    \lim_{T\rightarrow\infty}\left[\frac{1}{n}\sum_{i=1}^{n}\mathbb{E}\left\{\gamma^\top\hat{\Sigma}_{i}\gamma-\exp(\bx_{i}^\top
                    \beta)\right\}^{2}-\frac{1}{n}\sum_{i=1}^{n}\mathbb{E}\left\{\gamma^\top
                    S_{i}^{*}\gamma-\exp(\bx_{i}^\top \beta)\right\}^{2}\right]=0
                    \end{equation}
                    $$
                    $$ \scriptsize
                    \begin{equation}
                    \Leftrightarrow \quad \mathbb{E}\|\hat{\Sigma}_{i}- S_{i}^{*}\|^{2} \rightarrow 0, \quad \text{for
                    } i=1,\dots,n.
                    \end{equation}
                    $$
                </div>
                <p>Our Covariate-dependent Shrinkage CAP (CS-CAP) is optimal</p>
            </section>

            <section>
                <h2>Simu: new CS-CAP vs L-W and CAP</h2>
                <img src="Media/CAP/shrinkage_cap_simu.png" alt="" width="80%">
            </section>

            <section>
                <h2>Analysis of ADNI Data</h2>
                <ul>
                    <li>The Alzheimer’s Disease Neuroimaging Initiative (ADNI): launched in 2003, for studying ADRD</li>
                    <li>Alzheimer's Disease Related Dementias (ADRD) affects more than 6 million US people, and 55
                        million worldwide</li>
                    <li>No known* treatment to stop or prevent ADRD</li>
                    <li>fMRI and brain connectivity likely to be interrupted prior to dementia</li>
                    <li>APOE-$\varepsilon$4 gene, strong risk factor and potential treatment target</li>
                </ul>
            </section>

            <section>
                <h2>Covariate Implicated Components</h2>
                <img src="Media/CAP/high_dim_cap_adni_table.png" alt="" width="80%">
                <br>
                <ul>
                    <li>First two components related to age and sex</li>
                    <li>Last C3 predicted by age and APOE-$\varepsilon$4 gene</li>
                </ul>
            </section>

            <section>
                <h2>C3: APOE Areas Found by CS-CAP</h2>
                <img src="Media/CAP/high_dim_cap_adni_plot.png" alt="" width="80%">
                <br>
                <p>Groups of regions with more or less connections predicted by APOE-$\varepsilon$4 (non)-carriers</p>
            </section>

            <section data-state="flaticon2">
                <h2>Discussion</h2>
                <ul>
                    <li>Regress <emph>PD</emph> matrices on vectors</li>
                    <li>Method to identify covariate-related (supervised) directions vs (unsupervised) PCA</li>
                    <li>Theorectical justifications</li>
                    <li>Papers: Biostatistics (<a href="https://doi.org/10.1093/biostatistics/kxz057">10.1093/biostatistics/kxz057</a>), EJS
                        (<a href="https://doi.org/10.1214/21-EJS1887">10.1214/21-EJS1887</a>), Brain Behv (<a
                            href="https://doi.org/10.1002/brb3.1942">10.1002/brb3.1942</a>)</li>
                    <li>R pkg: <highlight>cap</highlight> <img src="Media/Flaticon/baseball-cap.png" style="width: 5%" alt=""></li>
                </ul>
                <style>
                    .flaticon2 credit:after {
                        content: "Image source: flaticon.com";
                    }
                </style>
            </section>




            <section>
                <!--               data-background="Media/Scene/barcelona.jpg" data-state="opaqueBackImg"-->
                <h1>Thank you!</h1>
                <br>
                <h2>Comments? Questions?</h2>
                <br>
                <br><br>

                <!--              <h2>Slides at:-->
                <!--
                    <emph>bit.ly/XLICSA16</emph>
                </h2>
                <img src="./Media/QR/XLICSA16.png" alt="" width="30%">
                <h2>
                    OR:
-->
                <span style="color: #0072bd;  font-weight: bold;">Big</span><span style="color: crimson;  font-weight: bold;">Complex</span><span
                    style="color: #7e2f8e;  font-weight: bold;">Data</span><span style="color: grey;  font-weight: bold;">.com</span>
                <br>
                <br> or
                <span style="color: #0072bd;  font-weight: bold;">Brain</span><span style="color: crimson;  font-weight: bold;">Data</span><span
                    style="color: #7e2f8e;  font-weight: bold;">Science</span><span style="color: grey;  font-weight: bold;">.com</span>



                <!--                </h2>-->

            </section>

        </div>
        <!-- div slides -->
    </div>

    <script src="reveal.js_4_5/dist/reveal.js"></script>
    <script src="reveal.js_4_5/plugin/zoom/zoom.js"></script>
    <script src="reveal.js_4_5/plugin/notes/notes.js"></script>
    <script src="reveal.js_4_5/plugin/search/search.js"></script>
    <script src="reveal.js_4_5/plugin/markdown/markdown.js"></script>
    <script src="reveal.js_4_5/plugin/highlight/highlight.js"></script>
    <script src="reveal.js_4_5/plugin/math/math.js"></script>

    <script>
        // Also available as an ES module, see:
        // https://revealjs.com/initialization/
        Reveal.initialize({
            controls: false,
            progress: true,
            center: true,
            hash: true,
            transition: 'slide', // none/fade/slide/convex/concave/zoom

            width: 1024,
            height: 768,

            slideNumber: 'c/t',

            pdfSeparateFragments: false,

            math: {
                // mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
                config: 'TeX-AMS_HTML-full',
                TeX: {
                    Macros: {
                        R: '\\mathbb{R}',
                        RR: "\\mathbf{R}",
                        real: "\\mathbb{R}",
                        set: ['\\left\\{#1 \\; ; \\; #2\\right\\}', 2]
                    }
                }
            },

            mathjax3: {
                mathjax: "https://cdnjs.cloudflare.com/ajax/libs/mathjax/3.2.2/es5/tex-svg.min.js", // 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js',
                tex: {
                    inlineMath: [['$', '$'], ['\\(', '\\)']],
                    macros: {
                        R: '\\mathbb{R}',
                        RR: "\\mathbf{R}",
                        real: "\\mathbb{R}",
                        bx: "x",
                        set: ['\\left\\{#1 \\; ; \\; #2\\right\\}', 2]
                    }
                },
                options: {
                    skipHtmlTags: ['script', 'noscript', 'style', 'textarea', 'pre']
                },
            },

            //anything to load other libraries
            anything: [{
                className: "coverflow",
                initialize: (function (container, options) {
                    if (typeof container.coverflow === 'function') {
                        container.coverflow()
                    }
                })
            }],

            // Learn about plugins: https://revealjs.com/plugins/
            plugins: [RevealZoom, RevealNotes, RevealSearch, RevealMarkdown, RevealMath.MathJax3, RevealHighlight]
        });

    </script>

    <!-- Coverflow event listeners -->
    <script src="JSLibrary/coverflowjs/coverflow.min.js"></script>

    <script>
        Reveal.addEventListener('fragmentshown', function (event) {
            jQuery(event.fragment).closest('.ui-coverflow').coverflow('next');
        });
        Reveal.addEventListener('fragmenthidden', function (event) {
            jQuery(event.fragment).closest('.ui-coverflow').coverflow('prev');
        });
        Reveal.addEventListener('slidechanged', function (event) {
            jQuery(event.currentSlide).find('.coverflow').coverflow();
            jQuery(event.currentSlide).find('.coverflow').coverflow('select', 0);
            jQuery(event.currentSlide).find('.coverflow').find('.fragment')
                .removeClass('visible').removeClass('currentSlide');
        });
        // $(function(){ $(Reveal.getCurrentSlide()).find('.coverflow').coverflow()  });
        //        setTimeout(function() {
        //            $(Reveal.getCurrentSlide()).find('.coverflow').coverflow()
        //        }, 15000);
        //        Reveal.addEventListener('ready', function(event) {
        //           let elements =  document.getElementsByClassName('coverflow')
        //           for (let j=0; j<elements.length; j++) {
        //               elements[j].coverflow();
        //           }
        //        });

    </script>



</body>

</html>